### PR TITLE
Adding section around Vespa in the vector DB blog

### PR DESCRIPTION
**Pull request by davidlago**
Added a section on how to run the examples with Vespa. Depends on https://github.com/dblock/vectordb-hello-world/pull/1 to be merged first as it links to it.

### CODE REVIEW

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: vector search -> approximate nearest neighbor search

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Since it's a full sentence finish with a period. (structure.)

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Let's be consistent with the above. We use full sentences, so just say "Deploy using the configuration API."

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: we can then -> setup the client.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Period.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (dblock) [inline _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Period.

**Review by dblock — APPROVED** [review]
Rockstar! I'll wait for the code PR to be merged before merging this. I have some *super* nitpics if you care about my OCD.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated.

File: _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md
Comment (davidlago) [reply _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md]: Updated.

**Pull request merged**
